### PR TITLE
OpenStack support for etcd-manager

### PR DESCRIPTION
**Pull request by zetaab**
Thanks to @drekle from original PR https://github.com/kopeio/etcd-manager/pull/195 

I took that one and fixed it.

Screenshot 2019-08-13 at 10 28 53

@justinsb could you please review this

### CODE REVIEW

File: pkg/volumes/openstack/volumes.go [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+		if err != nil {
+			return fmt.Errorf("Failed to retrieve server information from cloud: %v", err)
+		}
+		ip, err := GetServerFixedIP(extendedServer.Addresses, extendedServer.Name)
+		if err != nil {
+			return fmt.Errorf("error querying InternalIP from name: %v", err)
+		}
+		stack.internalIP = net.ParseIP(ip)
+		stack.zone = extendedServer.AvailabilityZone
+		glog.Infof("Found internalIP=%q and zone=%q", stack.internalIP, stack.zone)
```
Comment (drekle) [inline pkg/volumes/openstack/volumes.go]: maybe we should update to klog if we have not.

**Comment by drekle** [comment]
Thanks for picking this up @zetaab 
👍

File: pkg/volumes/openstack/volumes.go (hunk 1 above)
Comment (justinsb) [reply pkg/volumes/openstack/volumes.go]: Yes - would be great if someone did that, but not a blocker for this PR.

I'll get to it if nobody else beats me to it :-)

**Comment by justinsb** [comment]
Thanks @zetaab 

/approve
/lgtm

**Pull request merged**